rfcs/0021-ckb-address-format/0021-ckb-address-format.md

```diff
@@ -21,15 +21,32 @@ To generate a CKB address, we firstly encode lock script to bytes array, name *p
 
 There are several methods to convert lock script into payload bytes array. We use 1 byte to identify the payload format.
 
-| format type |                   description                  |
-|:-----------:|------------------------------------------------|
-|  0x01       | short version for locks with popular code_hash |
-|  0x02       | full version with hash_type = "Data"           |
-|  0x04       | full version with hash_type = "Type"           |
+| format type |                   description                        |
+|:-----------:|------------------------------------------------------|
+|  0x00       | full version identifies the hash_type and vm_version |
```

Comment:
This `format type` just specifies the `hash_type` while the `hash_type` sets `vm_version`, `type script`, `data hash` or something. Thus the `vm_version` should not be listed here or we have to update the table anytime the `hash_type` has updates.